### DISCUSSION TITLE
Reassign an existing variable?

### DISCUSSION THREAD

**Discussion by Nicryc** [starship/starship#5966] [Q&A] [answered]
Is it possible to modify an existing variable in a module?

For example, in the module `git_status` there is a variable named `$all_status` that contains `$conflicted$stashed$deleted$renamed$modified$typechanged$staged$untracked`.
I want to redefine this variable by changing order of the various variable used.

**Comment by davidkna** [comment, marked as answer]
`$all_status` cannot be redefined, but you can replace `$all_status` in the module format string directly, e. g. by setting it to `'([\[$conflicted$stashed$deleted$renamed$modified$typechanged$staged$untracked$ahead_behind\]]($style) )'` instead of the default `'([\[$all_status$ahead_behind\]]($style) )'`.
